deploy/warehouse/target/data.sql

```diff
@@ -16,9 +26,16 @@ insert into warehouse.organism (lineage, identifiers, details)
         ('Influenza.C',             'NCBITAXON => 11552',       null),
         ('Measles',                 'NCBITAXON => 11234',       '{"report_to_public_health": true}'),
         ('Mumps',                   'NCBITAXON => 1979165',     '{"report_to_public_health": true}'),
+        ('Mycoplasma_pneumoniae',   null,                       null),
+        ('Parainfluenza.1_2',       null,                       null),
+        ('Parainfluenza.3_4',       null,                       null),
```

Comment:
Ditto different species for parainfluenza 1, 2, 3, and 4.

Reply:
So, skip these along with Coronavirus for now?

Reply:
Yep.